lab/requirements.md

```diff
@@ -0,0 +1,51 @@
+# OPI Lab Requirements
+
+## Access
+
+- SSH connection is available through VPN.
+- Emergency console can be enabled for remote access in case of issue.
+
+## Backup
+
+- All critical servers must be backed up.
+- Restoration of critical servers must be done within a couple of hours.
+- Hosting provider must be trained to do backup/restore.
+
+## Capacity
+
+- Wait queue for automatic tests should be less than 1 hour.
+- Sufficient excess capacity so that manual workloads does not affect automatic tests too much.
```

Comment:
I'd remove this as we've re-focused this on automatic tests only.

Reply:
Yes makes sense to remove this for now, until we introduce interoperability tests and/or multi-vendor development tests.